### PR TITLE
Docs Update

### PR DESCRIPTION
**Pull request by wlan0**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,61 +1,97 @@
...
+
+### Architecture
+
+DirectPV is designed to be lightweight and scalable to 10s of 1000s of drives. It is made up of three components:
+
+- Controller
+- Node Driver
+- UI
+
+![DirectPV Architecture](./directpv_architecture.png)
```
Comment (sinhaashish) [inline README.md:18]: copy this image to `docs/screenshots` and replace with the complete path

File: README.md [hunk 2]
```diff
@@ -1,61 +1,97 @@
 DirectPV
 ----------
 
-![build](https://github.com/minio/directpv/workflows/Go/badge.svg) ![license](https://img.shields.io/badge/license-AGPL%20V3-blue) 
-
-DirectPV is a [CSI driver](https://kubernetes-csi.github.io/docs/introduction.html) for dynamically provisioning direct persistent volumes. 
+DirectPV is a CSI driver for [Direct Attached Storage](https://en.wikipedia.org/wiki/Direct-attached_storage). In a simpler sense, it is a distributed persistent volume manager, and not a storage system like SAN or NAS. It is useful to *discover, format, mount, schedule and monitor* drives across servers. Since Kubernetes `hostPath` and `local` PV are statically provisioned and limited in functionality, DirectPV was created to address this limitation.
```
Comment (ravindk89) [inline README.md:5]:
```suggestion

DirectPV was previously named DirectCSI and is available as an equivalent binary starting with the 2.0.0 release. Kubernetes deployments using older DirectCSI versions should install DirectPV as part of their upgrade to 2.0.0+. See
the [Upgrades](#upgrade) for instructions on upgrading to DirectPV.

```

Nice to have this up top to catch users up. Looking at the releases,

File: README.md [hunk 3]
```diff
@@ -1,61 +1,97 @@
...
 ```
 
-For more information, please visit our [documentation](./docs/index.md).
+For air-gapped setups and advanced installations, please refer to the [Installation guide](./docs/installation.md).
 
-### How is it different from LocalVolume provisioner?
+### Upgrade
 
-[LocalVolume](https://kubernetes.io/blog/2019/04/04/kubernetes-1.14-local-persistent-volumes-ga/) provisioner also allows direct access to the storage medium. However, it requires manual management of drives and volumes - i.e. it does not support dynamic provisioning of volumes, storage management or dynamic scheduling of pods on nodes with available volumes. 
+DirectPV version upgrades are seameless and transparent. Simply uninstall an existing version of directpv and install with a newer version to upgrade.
```
Comment (ravindk89) [inline README.md:61]:
```suggestion
DirectPV version upgrades are seameless and transparent. Simply uninstall an existing version of directpv and install with a newer version to upgrade. For DirectCSI installations, DirectPV provides a binary replacement for DirectCSI starting with 2.0.0. 
```

File: README.md [hunk 4]
```diff
@@ -1,61 +1,97 @@
...
 
-DirectPV on the other hand, supports dynamic provisioning of volumes, performs allocation of volumes based on storage capacity, and schedules pods to run on nodes which have most capacity available. 
+```
+# Uninstall directpv
+kubectl directpv uninstall
```
Comment (ravindk89) [inline README.md:65]:
```suggestion
# Uncomment to remove an older DirectCSI installation
# kubectl directcsi uninstall

# Uninstall directpv
kubectl directpv uninstall 
```

**Review by ravindk89 — CHANGES_REQUESTED** [review]
A few suggestions around making sure users know that:

1) We are migrating from DirectCSI -> DirectPV and
2) Its a binary replacement starting with 2.0.0

File: README.md (hunk 1 above)
Comment (wlan0) [reply README.md:18]: we can call it `docs/images`, since they are not really screenshots?

File: README.md (hunk 2 above)
Comment (wlan0) [reply README.md:5]: We are not replacing direct-csi by directpv. We are instead launching directPV for the first time now. Think of  direct-csi as a temporary working name we were using so far

File: README.md (hunk 4 above)
Comment (wlan0) [reply README.md:65]: The uninstall is a required step even before calling it directpv. The uninstall is a  part of the regular upgrade path. No application pods will  be harmed during/after uninstall.

File: README.md (hunk 3 above)
Comment (wlan0) [reply README.md:61]: The upgrade procedure does not change when going from direct-csi to directpv

**Pull request merged**